docs/src/main/asciidoc/hibernate-panache.adoc

```diff
@@ -0,0 +1,1192 @@
+implementation("io.quarkus:quarkus-hibernate-panache")
+
+// Pick your database driver
+implementation("io.quarkus:quarkus-jdbc-postgresql")
+----
+
+You also have to configure the Hibernate processor, which is required:
+
+[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
+.pom.xml
+----
+<plugin>
+    <artifactId>maven-compiler-plugin</artifactId>
+    <configuration>
+        <!-- This setting is required for the annotation processor dependencies to be managed by Quarkus.
+             More information is available in Maven compiler plugin documentation:
+             https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#annotationProcessorPathsUseDepMgmt -->
+        <annotationProcessorPathsUseDepMgmt>true</annotationProcessorPathsUseDepMgmt>
+        <annotationProcessorPaths>
+            <path>
+                <groupId>org.hibernate.orm</groupId>
+                <artifactId>hibernate-processor</artifactId>
+                <!-- Note, no artifact version is required, it's managed by Quarkus.  -->
+            </path>
+            <!-- other processors that may be required by your app -->
+        </annotationProcessorPaths>
+        <!-- Other compiler plugin configuration options -->
+    </configuration>
+</plugin>
+----
+
+[source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
+.build.gradle
+----
+// Enforce the version management of your annotation processor dependencies,
+// so that there's no need to define an explicit version of the hibernate-processor
+annotationProcessor enforcedPlatform("${quarkusPlatformGroupId}:${quarkusPlatformArtifactId}:${quarkusPlatformVersion}")
+annotationProcessor 'org.hibernate.orm:hibernate-processor'
+----
+
+TIP: You can use any of the xref:hibernate-orm.adoc#setting-up-and-configuring-hibernate-orm[supported JDBC drivers].
+
+Make sure you configure your datasource in your `application.properties` (although you don't have to in DEV
+mode: if you don't, a dev service will be provided for you):
+
+```properties
+%prod.quarkus.datasource.username = hibernate
+%prod.quarkus.datasource.password = hibernate
+%prod.quarkus.datasource.jdbc.url = jdbc:postgresql://localhost:5432/hibernate_db
+```
+
+== Our first entity
+
+What this guide will focus on is the basics of how to use Hibernate with Panache, so let's start with how to
+create an entity:
+
+```java
+import io.quarkus.hibernate.panache.PanacheEntity;
+import jakarta.persistence.Entity;
+
+import java.time.LocalDate;
+
+@Entity
+public class Cat extends PanacheEntity {
+    public String name;
+    public LocalDate birth;
+    public Breed breed;
+
+    public enum Breed {
+        CUTE, HAIRLESS;
+    }
+}
+```
+
+In order to map a Java class to a Database table, just:
+
+- Write a class
+- Annotate it with link:{jakarta-persistence-javadocs-url}/jakarta/persistence/entity[`@Entity`]
+- Make it extend `PanacheEntity`
+- Make its fields `public`
```

Comment:
Is it mandatory now?

Reply:
I don't think so, no, that's a simplification, but at this stage in the docs that's really the best description. Extending `PanacheEntity` is also not mandatory.

I should probably copy the "public field / getter" section from the other versions of Panache here to clear this up.